ActionButton.js

```diff
@@ -181,6 +181,9 @@ export default class ActionButton extends Component {
 
     return (
       React.Children.map(this.props.children, (button, index) => {
+        const length = this.props.children.length;
```

Comment:
@Ajackster that's what I meant. So if the button is positioned to the right, no button item should have the tooltip to the left side,?  So in that case, one way to do would be `const middle = (index == length -1)` and the rest are aligned to the right. I saw that in the new screenshot you posted, all tooltips are at the top of the button item. Is that something you would prefer using, instead of aligning the tooltip with respect to the index?